src/Pact/Native.hs

```diff
@@ -1587,6 +1589,51 @@ poseidonHackAChainDef = defGasRNative
         return $ toTerm $ poseidon intArgs
      | otherwise = argsError i as
 
+keccak256Def :: NativeDef
+keccak256Def = defGasRNative
+  "keccak256"
```

Comment:
a more descriptive name would be appreciated, as tedious as that is to change at this point. Is this a hash? encode? needs to be in the name, docs, and reflected in the argument names.